app/renderer/actions/Inspector.js

```diff
@@ -106,7 +89,7 @@ const findElement = _.debounce(async function (strategyMap, dispatch, getState,
     let {elementId, variableName, variableType} = await callClientMethod({
       strategy,
       selector,
-    });
+    })(dispatch, getState);
```

Comment:
Would it be possible to make this call more explicit? To me the `await f()()` does not seem very readable

Reply:
This is unfortunately the pattern we use everywhere anytime we call an action from another action, since we use redux's thunk model

Reply:
perhaps we could just do something like

```
const action = callClientMethod({
      strategy,
      selector,
    });
await action(dispatch, getState);
```

to improve the readability?